src/pages/Page.js

```diff
@@ -1,6 +1,8 @@
 import PropTypes from 'prop-types'
 import { Component } from 'react'
 
+// TODO: Delete file
```

Comment:
I assume there's a reason that this hasn't been deleted yet. I think it'd be good to add this reason to the comment, just so that's it's clear when this can be deleted.

Reply:
Good catch - this file was where the page state was being managed and now that all components have been migrated away from this pattern it is no longer needed